processor/src/main/java/org/mapstruct/ap/internal/util/Message.java

```diff
@@ -61,6 +61,7 @@
     PROPERTYMAPPING_NO_PRESENCE_CHECKER_FOR_SOURCE_TYPE( "Using custom source value presence checking strategy, but no presence checker found for %s in source type." ),
     PROPERTYMAPPING_NO_READ_ACCESSOR_FOR_TARGET_TYPE( "No read accessor found for property \"%s\" in target type." ),
     PROPERTYMAPPING_NO_WRITE_ACCESSOR_FOR_TARGET_TYPE( "No write accessor found for property \"%s\" in target type." ),
+    PROPERTYMAPPING_WHITESPACE_TRIMMED( "The property name \"%s\" has whitespace that was trimmed.", Diagnostic.Kind.WARNING ),
```

Comment:
Maybe the message `The property named \"%s\" has whitespaces, using trimmed property \"%s\" instead.` is more appropriate (similar to `PROPERTYMAPPING_INVALID_PROPERTY_NAME`). And the users will get the actual property that is being used. Not sure if it too much